classes/class-twenty-twenty-one-dark-mode.php

```diff
@@ -164,8 +164,8 @@ public function customizer_controls( $wp_customize ) {
 			array(
 				'type'            => 'checkbox',
 				'section'         => 'colors',
-				'label'           => esc_html__( 'Respect visitor\'s device dark mode settings', 'twentytwentyone' ),
-				'description'     => __( 'Dark mode is a device setting. If a visitor to your site requests it, your site will be shown with a dark background and light text.', 'twentytwentyone' ),
+				'label'           => esc_html__( 'Dark Mode', 'twentytwentyone' ),
```

Comment:
Clicking the checkbox doesn't enable dark-mode... Dark mode will only be automatically enabled if the visitor is using dark-mode in their OS settings.
So we can't call this setting "dark mode", it really is about respecting the visitor's OS settings, and allowing them to switch if they want.

Reply:
My understanding and what I saw is that when clicking the checkbox, the dark mode button appears, and user can click to enable it.
Is that right?

Reply:
It does both and that is part of the difficulty

Reply:
Yes. But if the checkbox is labeled "Dark Mode", it gives the impression that checking the checkbox will enable dark-mode, which is not what happens :smile: